packages/discord.js/typings/index.d.ts

```diff
@@ -4911,9 +4920,9 @@ export interface ClientEvents {
     oldGuildScheduledEvent: GuildScheduledEvent | null,
```

Comment:
```suggestion
 oldGuildScheduledEvent: GuildScheduledEvent | PartialGuildScheduledEvent | null,
```

This also can be partial, when cached structure was partial